### DISCUSSION TITLE
get_orientation_matrix, get_orientation_quaternion

### DISCUSSION THREAD

**Discussion by Simon-Steinmann** [cyberbotics/webots#3909] [Ideas]
**Is your feature request related to a problem? Please describe.**
The supervisor function 'get_orientation' currently returns a 1x9 list instead of a 3x3 list, which one would expect, since it represents the rotation matrix. Instead of changing this function, I suggest adding two new ones, as changing the existing function could break existing code.

**Describe the solution you'd like**
Add two new functions:
get_orientation_matrix() which returns a 3x3 list

get_orientation_quaternion() which returns a 1x4 list with the orientation expressed as quaternion. Preferably in [x, y, z, w] notation, as this is most commonly used in ROS and thus a majority of robotic projects and applications

**Comment by omichel** [comment]
@Simon-Steinmann: Please go ahead and open a pull request to propose this new feature. When introducing new API functions, we have to follow these [guidelines](https://github.com/cyberbotics/webots/wiki/Adding-New-Node-and-API-Function#adding-a-new-api-function) to ensure the new functions are implemented in all languages, all platforms, are fully documented and tested. We will be happy to review your contribution.
